website/common/script/content/quests.js

```diff
@@ -14,1850 +14,1815 @@ const CURRENT_EVENT = find(
 );
 const userCanOwnQuestCategories = USER_CAN_OWN_QUEST_CATEGORIES;
 
+// separatihg & sorting quests by type --
+// generic unlockable, chain, masterclasser, seasonal, pet, potion, time travelers, world
+// with intent of eventually refactoring
```

Comment:
Maybe we leave this for a separate refactoring PR, then? It'd be good to be able to review the functionally relevant additions without having to wade through all the cut-paste intended for later work.

Reply:
Because you have the refactoring PR out now, would it be OK to skip the alphabetization changes in quests.js for this PR? It's not necessary to the fix and it's going to introduce conflicts with the refactor.